main.go

```diff
@@ -208,11 +212,40 @@ func run(state overseer.State) {
 		handlers.SetArchiveMaxTimeout(*archiveTimeout)
 	}
 
+	// Build include and exclude detector filter sets.
+	var includeDetectorTypes, excludeDetectorTypes map[detectorspb.DetectorType]struct{}
+	{
+		includeList, err := config.ParseDetectors(*includeDetectors)
+		if err != nil {
+			// Exit if there was an error to inform the user of the misconfiguration.
+			logger.Error(err, "invalid detector configuration")
```

Comment:
__optional suggestion__: For ease of potentially figuring out later, it might make sense to change the error messages here to something like "invalid include list detector configuration" and "invalid exclude list detector configuration" to make it easier to know which was misconfigured.